src/gpodder/model.py

```diff
@@ -305,6 +332,19 @@ def from_podcastparser_entry(cls, entry, channel):
         episode.chapters = None
         if entry.get("chapters"):
             episode.chapters = json.dumps(entry["chapters"])
+        
+        if entry.get('number'):
+            episode.episode_num = entry['number']
```

Comment:
`episode.episode_num = entry.get('number', 0)`

Reply:
same for `season` please

Reply:
resolved in [aa967bc](https://github.com/gpodder/gpodder/pull/1432/commits/aa967bc44bae34c9b31831ec1e067a42481dcc7d)